server.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/acm-uiuc/arbor/server"
+	"github.com/acm-uiuc/groot-api-gateway/config"
```

Comment:
So when this is merged there will be a slightly different process in how you install groot. you still symlink the api gateway to [gopath]/github.com/acm-uiuc/groot-api-gateway, but now you can just to a go install github.com/acm-uiuc/arbor/* to get all the packages